coreneuron/io/core2nrn_data_return.cpp

```diff
@@ -546,7 +546,8 @@ void core2nrn_tqueue(NrnThread& nt) {
     }
     // TQitems from binq_
     for (q = tqe->binq_->first(); q; q = tqe->binq_->next(q)) {
-        assert(core2nrn_tqueue_item(q, sewm, nt) == false);
+        bool const result = core2nrn_tqueue_item(q, sewm, nt);
+        assert(result == false);
```

Comment:
why did you not replace here also with `nrn_assert`?

Reply:
No strong reason, I think just that the header wasn't already included / it wasn't already used in this file...